lib/src/git.rs

```diff
@@ -232,7 +245,7 @@ pub fn export_changes(
             git_ref.delete()?;
         }
     }
-    Ok(())
+    Ok(exported_view)
```

Comment:
Here we build a separate `exported_view` in order to minimize the serialized data? After "git: update our record of Git branches on export", `mut_repo.view()` should also have a valid data.

Just wanna confirm if I understand things correctly.

Reply:
Yes, good point! I had not even thought of that. I think you're right that we should be able to remove this extra `exported_view` at the end of this series. I'll look into adding a commit for that.

Reply:
Wait, no, `mut_repo.view()`'s branches will not be the same what we exported to Git if any of the branches had conflicts. When there are branches with conflicts, the `exported_view` will have those branches left at an older state. That way, we try to export that branch again on subsequent exports, until it no longer has a conflict.

However, `mut_repo.view()`'s Git refs should match the export state, so I think we should be able to compare the exported Git refs to current local branches. I'll have to think a bit more about that.